README.md

```diff
@@ -92,146 +93,52 @@ official compatibility chart below.
-- `--max-watch-seconds=seconds`: Raise a timeout error if it takes longer than _seconds_ for any
+- `--global-timeout=seconds`: Raise a timeout error if it takes longer than _seconds_ for any
 resource to deploy.
-- `--selector`: Instructs kubernetes-deploy to only prune resources which match the specified label selector, such as `environment=staging`. If you use this option, all resource templates must specify matching labels. See [Sharing a namespace](#sharing-a-namespace) below.
+- `--selector`: Instructs krane to only prune resources which match the specified label selector, such as `environment=staging`. If you use this option, all resource templates must specify matching labels. See [Sharing a namespace](#sharing-a-namespace) below.
+- `--no-verify-result`: Skip verification that workloads correctly restarted.
+- `--protected-namespaces=default kube-system kube-public`: Fail validation if a deploy is targeted at a protected namespace.
+- `--verbose-log-prefix`: Add [context][namespace] to the log prefix
+
 
 > **NOTICE**: Deploy Secret resources at your own risk. Although we will fix any reported leak vectors with urgency, we cannot guarantee that sensitive information will never be logged.
 
 ### Sharing a namespace
 
-By default, kubernetes-deploy will prune any resources in the target namespace which have the `kubectl.kubernetes.io/last-applied-configuration` annotation and are not a result of the current deployment process, on the assumption that there is a one-to-one relationship between application deployment and namespace, and that a deployment provisions all relevant resources in the namespace.
+By default, krane will prune any resources in the target namespace which have the `kubectl.kubernetes.io/last-applied-configuration` annotation and are not a result of the current deployment process, on the assumption that there is a one-to-one relationship between application deployment and namespace, and that a deployment provisions all relevant resources in the namespace.
 
 If you need to, you may specify `--no-prune` to disable all pruning behaviour, but this is not recommended.
 
-If you need to share a namespace with resources which are managed by other tools or indeed other kubernetes-deploy deployments, you can supply the `--selector` option, such that only resources with labels matching the selector are considered for pruning.
+If you need to share a namespace with resources which are managed by other tools or indeed other krane deployments, you can supply the `--selector` option, such that only resources with labels matching the selector are considered for pruning.
 
 ### Using templates and variables
 
 Each app's templates are expected to be stored in a single directory. If this is not the case, you can create a directory containing symlinks to the templates. The recommended location for app's deploy directory is `{app root}/config/deploy/{env}`, but this is completely configurable.
 
```

Comment:
This is not strictly true anymore (though it is definitely a worth recommendation). Also, the header of this section mentions variables, but that's not what we talk about in the paragraph